libipl/p10/ipl0.C

```diff
@@ -306,6 +306,12 @@ static int ipl_updatehwmodel(void)
 
 	update_hwas_state(boot_file_absent);
 
+	if(!boot_file_absent) {
```

Comment:
Assuming boot file present from beginning why should we set state to unusable?

Reply:
Boot file is present indicates genesis/regulra boot -- which does cfam reset and resets to default value , not usable, incase of host triggred reboot , need to set status back to defult( no cfam reset in this path)